docs/src/templates/includes/components/core/menu/je-menu-header.hbs

```diff
@@ -0,0 +1,21 @@
+<header class="c-card c-card--outline c-card--outline--aboveMid c-card--rounded--large c-menuHeader c-mediaElement c-mediaElement--fullstack">
+    <img
+        class="c-mediaElement-img c-mediaElement-img--outlined c-mediaElement-img--negativeTop"
+        src="{{baseUrl}}/assets/img/examples/restaurant-logo.gif"
+        alt="Fozzie Kebab"
+        itemprop="image" />
+    <div class="c-mediaElement-content">
+        <h1 class="text-center u-noSpacing">Fozzie Kebab</h1>
+        <p class="c-badge c-badge--light c-badge--large text-center u-noSpacing">Kebab <span>•</span> Pizza</p>
+        <a class="l-centered u-noUnderline u-spacingTop" href="#">
+            {{> je-star-ratings this }}
+            {{#if star-rating-count}}<strong class="u-spacingLeft">{{star-rating-count}}</strong>{{/if}}
+        </a>
+        {{> (lookup . 'status') }}
```

Comment:
How does this work?  Guessing it links to the relevant file based on the status value?

Reply:
Yeah, it does a dynamic partial lookup based on the variable passed in.

Reply:
Clever – didn't know this was a default handlebars helper

Reply:
Neither did I [until I read this](https://handlebarsjs.com/partials.html#dynamic-partials) :)